### PR TITLE
Retrieve userId from SecurityRequestContext

### PR DESCRIPTION
**Pull request by gokulavasan**
For ConsoleSettingsHandler, get the user id from the SecurityRequestContext.

Also rename the endpoints from usersettings to consolesettings.

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ConsoleSettingsHttpHandler.java [hunk 1]
```diff
@@ -61,7 +62,11 @@ public ConsoleSettingsHttpHandler(Authenticator authenticator, ConsoleSettingsSt
   @Path("/")
   @GET
   public void get(HttpRequest request, HttpResponder responder) throws Exception {
-    String userId = getAuthenticatedAccountId(request);
+    String userId = SecurityRequestContext.getUserId();
```
Comment (jwang47) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ConsoleSettingsHttpHandler.java]: I'd suggest changing `SecurityRequestContext.getUserId()` to return an `Optional`, and then it's clear when `userId` is missing. Then you can retrieve the userId as follows:

```
String userId = SecurityRequestContext.getUserId().or("");
```

**Comment by jwang47** [comment]
One comment.

**Comment by gokulavasan** [comment]
Bamboo : https://builds.cask.co/browse/CDAP-RBT101

**Comment by gokulavasan** [comment]
@jwang47 Updated the PR addressing your comment. Please take a look!

**Comment by jwang47** [comment]
LGTM

**Pull request merged**